src/MLJInterface.jl

```diff
@@ -82,15 +81,15 @@ MLJModelInterface.@mlj_model mutable struct LGBMRegressor <: MLJModelInterface.D
     # Metrics
     metric::Vector{String} = ["l2"]::(all(in.(_, (LGBM_METRICS, ))))
```

Comment:
Unfortunately defaulted values are required on the MLJInterface so here ["l2"] has been left where it is [""] in estimators.jl (which means a default for pre-defined objectives).
This applies to all the cases where we could use an empty string. 
![image](https://github.com/user-attachments/assets/f54bcade-1020-447e-868b-58fa161d51d2)